static/js/components/CourseList.js

```diff
@@ -33,17 +42,25 @@ class CourseList extends React.Component {
         </ul>;
       });
 
-      return <div key={program.id} className="course-list">
-        <ul className="course-list-header">
-          <li className="course-title">
-            {program.title}
-          </li>
-          <li className="course-status "/>
-          <li className="course-progress">
-            Progress
-          </li>
-        </ul>
-        {table}
+      return <div key={program.id + "_1"}>
+        <div key={program.id} className="course-list">
+          <ul className="course-list-header">
+            <li className="course-title">
+              {program.title}
+            </li>
+            <li className="course-status "/>
+            <li className="course-progress">
+              Progress
+            </li>
+          </ul>
+          {table}
+        </div>
+        <div className="clear-float">
+          <br/>
+          <p>You need to pass {totalCourses - coursesPassed} more courses
```

Comment:
This text is per program but `coursesPassed` will keep incrementing for every program and course. It seems like you should define these two inside the `map` function and not the `render` function